src/components/DistanceRequest/index.js

```diff
@@ -130,6 +143,21 @@ function DistanceRequest({transactionID, report, transaction, route, isEditingRe
         Navigation.navigate(isEditingRequest ? ROUTES.MONEY_REQUEST_EDIT_WAYPOINT.getRoute(report.reportID, transactionID, index) : ROUTES.MONEY_REQUEST_WAYPOINT.getRoute('request', index));
     };
 
+    const getError = () => {
+        // get route error if available else show the invalid number of waypoints error
```

Comment:
Comments should start with a capital letter and end with a period (it should be a normal sentence). Please update this and the comment below as well.